app/Http/Controllers/Controller.php

```diff
@@ -10,9 +10,4 @@
 class Controller extends BaseController
 {
     use AuthorizesRequests, DispatchesJobs, ValidatesRequests;
-
-    public function index()
```

Comment:
đoạn này cứ trả về code cũ đi

Reply:
Đoạn này ban đầu nó k có hàm index trong đây a
Vả lại nếu gọi hàm index trong Controller thì em k gọi được hàm index trong CourseController vì CourseController extend Controller.
và xảy ra lỗi này ạ _`Declaration of App\Http\Controllers\CourseController::index(Illuminate\Http\Request $request) must be compatible with App\Http\Controllers\Controller::index()`_